src/styles/paststyle/card.scss

```diff
@@ -0,0 +1,99 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin-top: 0;
```

Comment:
Don't set margin-top for elements, if you need to reset margins for paragraphs - do it in reset.css file